README.md

```diff
@@ -195,7 +195,7 @@ It's possible to set various elements of `yagna` configuration through environme
 - `YAGNA_API_URL`, base URL to `yagna` REST API, e.g. `http://localhost:7500`
 - `YAGNA_APPKEY`, `yagna` app key to be used, e.g. `a70facb9501d4528a77f25574ab0f12b`
 - `YAGNA_MARKET_URL`, URL to `yagna` market API, e.g. `http://localhost:7500/market-api/v1`
-- `YAGNA_NETWORK`, Ethereum network name for `yagna` to use, e.g. `rinkeby`
+- `YAGNA_PAYMENT_NETWORK`, Ethereum network name for `yagna` to use, e.g. `rinkeby`
```

Comment:
This might cause a looot of really-hard-to-detect trouble for people who use those env variables. You update `yapapi` and now your code runs on `rinkeby` instead of on `mainnet`.

I think we should keep `YAGNA_NETWORK` here (or maybe make both work).